ADR/adr-0010-namespace-metadata

```diff
@@ -25,6 +25,14 @@ We will apply the following labels to Stone Soup namespaces, to make them easier
 - `stonesoup.redhat.com/namespacetype: "user-workspace-data"` for User workspaces where Applications, Components, and so on are defined
 - `stonesoup.redhat.com/namespacetype: "user-deployments"` for the namespaces where GitOps deploys applications for users
 - `stonesoup.redhat.com/namespacetype: "user-builds"` for the namespaces where the Pipeline Service has PipelineRuns
+- `stonesoup.redhat.com/workspace_id: "b2dece58-a260-11ed-a8fc-0242ac120002"` a unique identifier for the instance of the service (user or workspace UUID)
```

Comment:
It probably shouldn't be a user UUID if ownership can be transferred.

Reply:
That's true - can we just provide a unique workspace ID?